lost/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,14 @@
         android:required="false" />
     <uses-feature android:name="android.hardware.wifi" />
 
-    <!-- Wow. This is insane. You require an empty tag. -->
-    <application />
+
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
Good point. I've removed them from the library and updated our README to document their requirement when using this API